Cargo.toml

```diff
@@ -4,6 +4,83 @@ cargo-features = []
 resolver = "2"
 members = ["cli", "lib", "lib/testutils", "lib/gen-protos"]
 
+[workspace.dependencies]
+anyhow = "1.0.72"
+assert_cmd = "2.0.8"
+assert_matches = "1.5.0"
+backoff = "0.4.0"
+blake2 = "0.10.6"
+byteorder = "1.4.3"
+bytes = "1.4.0"
+cargo_metadata = "0.17.0"
+clap = { version = "4.3.19", features = ["derive", "deprecated"] }
+clap_complete = "4.3.2"
+clap_mangen = "0.2.10"
+chrono = { version = "0.4.26", default-features = false, features = [
+    "std",
+    "clock",
+] }
+config = { version = "0.13.2", default-features = false, features = ["toml"] }
+criterion = "0.5.1"
+crossterm = { version = "0.26", default-features = false }
+digest = "0.10.7"
+dirs = "5.0.1"
+esl01-renderdag = "0.3.0"
+glob = "0.3.1"
+git2 = "0.17.2"
+hex = "0.4.3"
+itertools = "0.11.0"
+indexmap = "2.0.0"
+libc = { version = "0.2.147" }
+insta = { version = "1.31.0", features = ["filters"] }
```

Comment:
nit: It looks like this adds the "filters" feature to the lib crate's dependency. That should be harmless (and maybe even good, so we don't have to recompile it between the lib and cli crates). But delete it from `cli/Cargo.toml` in that case.

(This is very much related to Yuya's comment)